packages/terser/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@rollup/plugin-terser",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Generate minified bundle",
+  "license": "MIT",
+  "repository": {
+    "url": "rollup/plugins",
+    "directory": "packages/terser"
+  },
+  "author": "Peter Placzek <peter.placzek1996@gmail.com>",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/terser#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
+  "exports": {
+    "types": "./types/index.d.ts",
+    "import": "./dist/es/index.js",
```

Comment:
Is that right to use .js extension for es modules in commonjs package?

Reply:
The other plugins set the type to "module" like this: https://cdn.jsdelivr.net/npm/@rollup/plugin-alias@4.0.0/dist/es/package.json

Reply:
As far as i know, you only have to specify the `.cjs` or `.mjs` extension if node does not know if it should read the file as cjs or esm module.